astropy/io/votable/tests/exception_test.py

```diff
@@ -17,7 +17,7 @@ def fail():
     except RuntimeError as e:
         assert "From here" in str(e)
     else:
-        assert False
+        raise AssertionError()
```

Comment:
I would rewrite this test to use `pytest.raises(RunTimeError, match="From here")` instead of the `try-except-else` block.

Reply:
SGTM.
Done!